### DISCUSSION TITLE
How to update the GPTK when you already have Whisky installed?

### DISCUSSION THREAD

**Discussion by luisclement** [Whisky-App/Whisky#345] [Q&A]
How to update the GPTK when you already have Whisky installed? Any hints even if manually? I had installed when GPTK beta 3 was out, very easy drag and drop and we now have beta 4 so how to update?

**Comment by clearluyy** [comment]
Click“whisky” which is left to your apple logo，then find" install"（third from the top),uninstall your current gptk version first, then drag beta 4 to there
Plus, the new version might update your gtpk version automatically
